src/routes/players/[userId]/+page.svelte

```diff
@@ -24,33 +25,111 @@
+            color: #ffffff; /* White header text color */
+        }
+
+        p {
+            text-align: center;
+            margin-bottom: 2rem;
+            color: #ffffff; /* White paragraph text color */
+        }
+
+        label {
+            font-size: 1rem;
+            margin-bottom: 0.5rem;
+            color: #ffffff; /* White label text color */
+        }
+
+        select {
+            font-size: 1rem;
+            background-color: #fff; /* White background */
+            border: 1px solid #ccc; /* Light gray border */
+            border-radius: 4px;
+            padding: 0.5rem;
+            margin-bottom: 1rem;
+        }
+
+        /* Style for the pie chart */
+        pie-chart {
+            width: 300px;
+            height: 300px;
+            margin-top: 2rem;
+        }
+    </style>
 </svelte:head>
 
-<h1>{data.userInfo.name}'s stats</h1>
-{#await data.userStats}
-    <h2>Weapons</h2>
-    <label for="select-active-weapon">Select weapon to view stats for</label>
-    <select id="select-active-weapon" bind:value={$activeWeapon}>
-        <option value="total">total</option>
-    </select>
-    <h3>Hits</h3>
-    <ShotVisualizer stats={activeWeaponStats} />
-{:then stats}
-    <h2>Weapons</h2>
-    <label for="select-active-weapon">Select weapon to view stats for</label>
-    <select id="select-active-weapon" bind:value={$activeWeapon}>
-        <option value="total">total</option>
-        {#each Object.keys(stats.weapons) as weaponId}
-            <option value={weaponId}>{weaponId}</option>
-        {/each}
-    </select>
-    <h3>Hits</h3>
-    <ShotVisualizer stats={activeWeaponStats} />
-{:catch error}
-    <h1>Weapons</h1>
-    <p>Seems like the stats API is having some trouble. Maybe check back later?</p>
-    <p>Error code: {error.code}</p>
-    <p>Details: {error.details}</p>
-{/await}
+<div class="container">
+    <h1>{data.userInfo.name}'s stats</h1>
+    {#await data.userStats}
+        <h2>Weapons</h2>
+        <label for="select-active-weapon">Select weapon to view stats for</label>
+        <select id="select-active-weapon" bind:value={$activeWeapon}>
+            <option value="total">total</option>
+        </select>
+        <h3>Hits</h3>
+        <ShotVisualizer stats={activeWeaponStats} />
+    {:then stats}
+        <h2>Weapons</h2>
+        <label for="select-active-weapon">Select weapon to view stats for</label>
+        <select id="select-active-weapon" bind:value={$activeWeapon}>
+            <option value="total">total</option>
+            {#each Object.keys(stats.weapons) as weaponId}
+                <option value={weaponId}>{weaponId}</option>
+            {/each}
+        </select>
+        <h3>Hits</h3>
+        <ShotVisualizer stats={activeWeaponStats} />
+        <!-- Placeholder for the pie chart -->
+        <svg class="pie-chart" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+            <!-- Draw the pie chart here -->
+        </svg>
```

Comment:
Move to seperate component?

Reply:
In the future just there as a place holder